internal/ingress/controller/controller.go

```diff
@@ -224,6 +224,10 @@ func (n *NGINXController) CheckIngress(ing *networking.Ingress) error {
 		return nil
 	}
 
+	if n.cfg.DisableCatchAll && ing.Spec.Backend != nil {
+		return fmt.Errorf("This deployment is trying to create a catch-all ingress while DisableCatchAll flag is set to true. Remove '.spec.backend' or set DisableCatchAll flag to false.")
```

Comment:
we should include namespace and the ingress name, also I'd follow the similar message as from the above logs

Reply:
oh never mind, this message will directly be exposed to the operator trying to deploy the ingress under consideration